packages/http/src/__tests__/http-prism-instance.spec.ts

```diff
@@ -172,6 +174,55 @@ describe('Http Client .process', () => {
           expect(result.output!.body).toEqual(reply);
         });
       });
+
+      describe('Prism user-agent header', () => {
+        it('adds user-agent header in the form of Prism/<<PRISM_VERSION>>', async () => {
+          const oasBaseUrl = 'http://example.com/api';
+
+          nock(oasBaseUrl)
+            .get('/pet')
+            .reply(function() {
+              expect(this.req.headers['user-agent']).toBe(`Prism/${prismVersion}`);
```

Comment:
Instead of doing the assertion here I would rather save the value in a variable and then asset it in a following and explicit `it` block — this should make the things easier to red.

```ts
describe("Prism user agent header", () => {
  describe("when the defaults are used", () => {
    it("shuold have Prism/version", () => {
      expect(savedvalue).toBe("Prism/version3.10");
    });
  });
});
```